src/Relude/List.hs

```diff
@@ -1,5 +1,3 @@
-{-# LANGUAGE Safe #-}
```

Comment:
@vrom911 @chshersh Is there a way to avoid removing this? It won't compile because of `Relude.Base` and `Relude.Numeric` not being "safe" 😟

Reply:
You can replace `Safe` with `Trustworthy` for now. It's totally okay to do so :+1: